projects/aca-content/src/lib/utils/aca-search-utils.ts

```diff
@@ -117,7 +117,7 @@ export function extractSearchedWordFromEncodedQuery(encodedQuery: string): strin
       ? userQuery
           .split('AND')
           .map((searchCondition) => {
-            const searchTerm = searchCondition.split('"')[1];
+            const searchTerm = searchCondition.includes('"') ? searchCondition.split('"')[1] : searchCondition.trim();
```

Comment:
please add unit tests to test it

Reply:
I think that needs some sort of regex here, for instance

```
const userQuery = `title:"Hello World" AND status:active AND "complex phrase"`;

const searchTerms = userQuery
  .split(/\s+AND\s+/i)
  .map((c) => {
    const match = c.match(/"([^"]+)"|(\S+)/);
    return match ? match[1] ?? match[2] : '';
  })
  .filter(Boolean);

console.log(searchTerms);
// ["Hello World", "status:active", "complex phrase"]
```